generator.yaml

```diff
@@ -161,3 +161,17 @@ resources:
       Tags:
         compare:
           is_ignored: true
+  OpenIDConnectProvider:
+    hooks:
+      delta_pre_compare:
+        code: compareTags(delta, a, b)
+    exceptions:
+      terminal_codes:
+        - InvalidInput 
+        - EntityAlreadyExists
```

Comment:
Is `EntityAlreadyExists` really a terminal error? Is it a "human input" error?

Reply:
This is intended to capture the case where the user has supplied a URL that is already associated with an OIDC Provider in their account.  From https://docs.aws.amazon.com/IAM/latest/APIReference/API_CreateOpenIDConnectProvider.html:

> You cannot register the same provider multiple times in a single AWS account. If you try to submit a URL that has already been used for an OpenID Connect provider in the AWS account, you will get an error.

My understanding is that anything error condition which is not "recoverable" should be considered "terminal".  If the user doesn't change the URL provided, then this error isn't recoverable.  Right?